src/scripts/unmapped_tables.py

```diff
@@ -68,7 +72,7 @@ def create_mapping_status_tables(
         'subject_id': curies_owned, 'subject_label': labels_owned, 'is_mapped': is_mapped, 'is_excluded': is_excluded,
         'is_deprecated': is_deprecated})
     if len(df) == 0:
-        return {'full': df, 'simple': df}
+        raise RuntimeError('Mapping status: No "owned terms" found in the ontology. Not expected.')
```

Comment:
@matentzn @hrshdhgd Previously, at this point if there were 0 terms found during this generation of mapping status / list of unmapped terms TSVs for the mapping progress monitor, I was not throwing an error. I realize now that this was a mistake. Now throws an error if no results, which is what I was getting before fixing the angle brackets issue.

Reply:
I'm assuming (and maybe I'm wrong) this is related to the Slack conversation we had regarding `omit.db`. Did recreating the file eliminate the need of handling `<>` ?

Reply:
Not prior to running this, but I will be re-running all of these.

I'm reluctant to remove this extra step using `remove_angle_brackets()` again though. I had been doing it in the past, and then I was no longer seeing URIs with angle brackets, so I removed this step, but then they appeared again. It affects performance very little, and I want to account for unknown unknowns / possible issues in the future.

One thing I can possibly see happening is a mismatch between `config/prefixes.csv` and the prefixes in the `metadata/%.yml` files. There's some redundancy there, and I think Nico is aware. We've had conversations before where we bounced between a single file vs multiple files, using an external registry, etc. So until we choose to use only one of those options I am wary of such a mismatch occurring.

Reply:
I think its ok to have this as a failsafe! But we do neet to remember when adding an ingest to add the appropriate prefixes to the various files we work with

@joeflack4 while you are working on adding medgen, can you write a 7 step guide on how to "add an ingest to Mondo INgest" and add it to the docs? This should include making sure the prefixes are up to date as well.